Dockerfile

```diff
@@ -193,7 +193,7 @@ RUN pip install scipy && \
     # PyTorch Audio
     apt-get install -y sox libsox-dev libsox-fmt-all && \
     pip install cffi && \
-    pip install git+git://github.com/pytorch/audio.git && \
+    pip install git+git://github.com/pytorch/audio.git@ce1f8aafe63f7d579c74ce54d2a85d452d55c881 && \
```

Comment:
Can you create a bug for this issue: https://b.corp.google.com/issues/new?component=451074&template=1187228

Just copy the good description you wrote from this PR to the bug and add a comment with a link to the bug in the Dockerfile so we know why they were pinned and when we should unpin them.

From the description, it seems like we should be able to unpin them soon so you can add the link to the bug as a todo:

# TODO(b/9999): Remove pin once pytorch audio version > X.Y is released.

Reply:
Thanks! Added bugs and TODOs. I kept them separate because the issues are likely to be resolved on different timelines with different resolutions.